documentation/modules/ROOT/pages/operations/kubernetes.adoc

```diff
@@ -0,0 +1,372 @@
+It can be created e.g. as follows:
+
+[source,bash]
+----
+$ cat << EOF | kubectl create -n debezium-example -f -
+apiVersion: v1
+kind: Service
+metadata:
+  name: mysql
+spec:
+  ports:
+  - port: 3306
+  selector:
+    app: mysql
+  clusterIP: None
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: mysql
+spec:
+  selector:
+    matchLabels:
+      app: mysql
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app: mysql
+    spec:
+      containers:
+      - image: quay.io/debezium/example-mysql:{debezium-docker-label}
+        name: mysql
+        env:
+        - name: MYSQL_ROOT_PASSWORD
+          value: debezium
+        - name: MYSQL_USER
+          value: mysqluser
+        - name: MYSQL_PASSWORD
+          value: mysqlpw
+        ports:
+        - containerPort: 3306
+          name: mysql
+EOF
+----
+
+== Deploying a {prodname} Connector
+
+To deploy a {prodname} connector, you need to deploy a Kafka Connect cluster with the required connector plug-in(s), before instantiating the actual connector itself.
+As the first step, a container image for Kafka Connect with the plug-in has to be created.
+If you  already have a container image built and available in the registry, you can skip this step.
+In this document, the MySQL connector will be used as an example.
+
+=== Creating Kafka Connect Cluster
+
+Again, we will use Strimzi for creating the Kafka Connect cluster.
+Strimzi also can be used for building and pushing the required container image for us.
+In fact, both tasks can be merged together and instructions for building the container image can be provided directly within the `KafkaConnect` object specification:
+
+[source,bash]
+----
+$ cat << EOF | kubectl create -n debezium-example -f -
+apiVersion: kafka.strimzi.io/v1beta2
+kind: KafkaConnect
+metadata:
+  name: debezium-connect-cluster
+  annotations:
+    strimzi.io/use-connector-resources: "true"
+spec:
+  version: 3.1.0
+  replicas: 1
+  bootstrapServers: debezium-cluster-kafka-bootstrap:9092
+  config:
+    group.id: connect-cluster
+    offset.storage.topic: connect-cluster-offsets
+    config.storage.topic: connect-cluster-configs
+    status.storage.topic: connect-cluster-status
+    # -1 means it will use the default replication factor configured in the broker
+    config.storage.replication.factor: -1
+    offset.storage.replication.factor: -1
+    status.storage.replication.factor: -1
+  build:
+    output:
+      type: docker
+      image: 10.110.154.103/debezium-connect-mysql:latest
+    plugins:
+      - name: debezium-mysql-connector
+        artifacts:
+          - type: tgz
+            url: https://repo1.maven.org/maven2/io/debezium/debezium-connector-mysql/{debezium-version}/debezium-connector-mysql-{debezium-version}-plugin.tar.gz
+            sha512sum: dff3e1f973ab203490d19337fac65e7545fbd70e3f22555e6ca74b3c734160140cbef5905b9f138f8b56616e0af5a41a8b2b7f09497d9cc62f64863017bdc088
```

Comment:
So this represents sort of a problem. I've used the `debezium-version` variable for always showing the current version in the rendered HTML page (this is sourced from _antora.yml_), but this will change the SHA of course. So we probably should add another variable for the SHA which needs updating alongside the version variable upon running the release // CC @jpechane, @Naros

Reply:
SHA is not mandatory, so another option is we can skip it to make it more easy - with a note that in real usage it should be used (same as with security - for simplicity we skip it)

Reply:
Ok, ocol. Thx!